tests/func/test_scm.py

```diff
@@ -34,27 +34,27 @@ def test_init_sub_dir(tmp_dir):
     assert scm.root_dir == os.fspath(tmp_dir)
 
 
-class TestSCMGit(TestGit):
-    def test_commit(self):
-        G = Git(self._root_dir)
-        G.add(["foo"])
-        G.commit("add")
-        self.assertIn("foo", self.git.git.ls_files())
-
-    def test_is_tracked(self):
-        foo = os.path.abspath(self.FOO)
-        G = Git(self._root_dir)
-        G.add([self.FOO, self.UNICODE])
-        self.assertTrue(G.is_tracked(foo))
-        self.assertTrue(G.is_tracked(self.FOO))
-        self.assertTrue(G.is_tracked(self.UNICODE))
-        G.commit("add")
-        self.assertTrue(G.is_tracked(foo))
-        self.assertTrue(G.is_tracked(self.FOO))
-        G.gitpython.repo.index.remove([self.FOO], working_tree=True)
-        self.assertFalse(G.is_tracked(foo))
-        self.assertFalse(G.is_tracked(self.FOO))
-        self.assertFalse(G.is_tracked("not-existing-file"))
+def test_commit(tmp_dir, scm):
+    tmp_dir.gen({"foo": "foo"})
+    scm.add(["foo"])
+    scm.commit("add")
+    assert scm.is_tracked("foo")
```

Comment:
```suggestion
    assert "foo" in scm.gitpython.git.ls_files()
```

(see my comments in the outdated thread)